### ISSUE TITLE
DNS issues with systemd-resolved

### ISSUE THREAD

**Issue by trytomakeyouprivate** [igo95862/bubblejail#92]
### Output of `bubblejail --version`

8.0.2

### Your distro name and version

Fedora 39

### Description

https://github.com/igo95862/bubblejail/issues/80

https://github.com/igo95862/bubblejail/issues/90

https://github.com/igo95862/bubblejail/issues/91

There is no DNS connectivity in the jails using the default configuration for Firefox.

```
bash-5.2$ ls -l /etc/resolv.conf
lrwxrwxrwx. 1 nfsnobody nfsnobody 39 Jan 28 17:06 /etc/resolv.conf -> ../run/systemd/resolve/stub-resolv.conf
```

**Comment by igo95862** [comment]
Can you test the d09981374c851e1bcfd5a6849304be1bb2a6b887 patch?

**Comment by igo95862** [comment]
Fixed with a6a2719410b95618a9c2dac1337a7f1a9a8d922f
